docs/outlook/spam-reporting.md

```diff
@@ -275,3 +278,7 @@ As you develop and test the integrated spam reporting feature in your add-in, be
 
 - [Office Add-ins manifest](../develop/add-in-manifests.md)
 - [Runtimes in Office Add-ins](../testing/runtimes.md)
+- [ReportPhishingCommandSurface extension point (preview)](/javascript/api/manifest/extensionpoint?view=outlook-js-preview&preserve-view=true#reportphishingcommandsurface-preview)
+- [Office.AddinCommands.EventCompletedOptions](/javascript/api/outlook/office.messageread?view=outlook-js-preview&preserve-view=true)
+- [Office.MessageRead.getAsFileAsync (preview)](/javascript/api/outlook/office.messageread?view=outlook-js-preview&preserve-view=true)
```

Comment:
will you be updating this link to include the anchor so it goes directly to the getAsFileAsync section of the page?

Reply:
Fixed.